src/black/__init__.py

```diff
@@ -501,10 +507,23 @@ def get_sources(
     force_exclude: Optional[Pattern[str]],
     report: "Report",
     stdin_filename: Optional[str],
-) -> Set[Path]:
+) -> Tuple[Set[Path], Path]:
     """Compute the set of files to be formatted."""
+    root, method = find_project_root(src)
+
+    if verbose:
+        if method:
+            out(
+                f"Identified `{root}` as project root containing a '{method}'.",
```

Comment:
```suggestion
                f"Identified `{root}` as project root containing a {method}.",
```
Looks better than the current output:
```
Identified `/Users/jelle/py/black` as project root containing a '.git directory'.
```